### PR TITLE
Refactor main components using Comp API part 1

### PR DESCRIPTION
**Pull request by samuveth**

### CODE REVIEW

File: src/components/Checkbox.vue [hunk 1]
```diff
@@ -9,24 +9,23 @@
 </template>
 
 <script>
+import { ref } from 'vue';
```
Comment (bonustrack) [inline src/components/Checkbox.vue:12]:
```suggestion
import { ref } from 'vue';

```

**Comment by samuveth** [comment]
This is now ready for testing @ChaituVR

File: src/views/Delegate.vue [hunk 2]
```diff
@@ -9,7 +9,7 @@
         <h1 v-if="loaded" v-text="$t('delegate.header')" />
       </div>
       <template v-if="loaded">
-        <Block :itle="$t('delegate.selectAddress')">
+        <Block :title="$t('delegate.selectAddress')">
```
Comment (ChaituVR) [inline src/views/Delegate.vue:12]: "Select address" is relevant? there is no dropdown right 🤔 ..  i think maybe it should be "Enter address" or "Enter an address to delegate"
image

**Review by ChaituVR — COMMENTED** [review]
LGTM, everything seems to work 👍

File: src/views/Delegate.vue (hunk 2 above)
Comment (samuveth) [reply src/views/Delegate.vue:12]: Ok I'll make a card

Comment (samuveth) [reply src/views/Delegate.vue:12]: Please approve if there's nothing else

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**